examples/user_interfaces/svg_tooltip.py

```diff
@@ -1,3 +1,6 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
 """
 SVG tooltip example
```

Comment:
Could you add a row of `===` above this to make the title sphinx-gallery compliant?